Ver 1kb (#201)
idaholab/TMAP8: doc/content/verification_and_validation/ver-1kb.md

# ver-1kb

# Henry’s Law Boundaries with No Volumetric Source

## General Case Description

Two enclosures are separated by a membrane that allows diffusion according to Henry’s law, with no volumetric source present. Enclosure 2 has twice the volume of Enclosure 1.

## Case Set Up

This verification problem is taken from [!cite](ambrosek2008verification).

> Over time, the pressures of T$_2$, which diffuses across the membrane in accordance with Henry’s law, will gradually equilibrate between the two enclosures.

Reviewer: You sometimes call the gas H_2 (test specification file, for example), sometimes T_2 (here and in the figures, for example). It's applicable to both, but please be consistent. I have a very slight preference for T_2 here.

Reviewer: @Lee01Atom please do not manually mark conversations as resolved, as I prefer to do it myself to keep track of what has been addressed and what hasn't. 
It is fine to have comments being marked as resolved automatically when you apply a suggestions, but please do not resolve them by hand. 
Thank you!

> The diffusion process in each of the two enclosures can be described by the following equations:

Reviewer: ```suggestion
The diffusion process in each of the two enclosures can be described by
```

\begin{equation}

> \frac{\partial C_1}{\partial t} = - \nabla D \nabla C_1

Reviewer: ```suggestion
\frac{\partial C_1}{\partial t} = \nabla D \nabla C_1,
```

\end{equation}

> 

Reviewer: ```suggestion
and
```

\begin{equation}

> \frac{\partial C_2}{\partial t} = - \nabla D \nabla C_2

Reviewer: ```suggestion
\frac{\partial C_2}{\partial t} = \nabla D \nabla C_2,
```

\end{equation}

> where $C_1$, $C_2$ represent the concentration fields in enclosures 1 and 2 respectively, and $D$ denotes the diffusivity.

Reviewer: ```suggestion
where $C_1$ and $C_2$ represent the concentration fields in enclosures 1 and 2 respectively, and $D$ denotes the diffusivity.
```

The concentration in Enclosure 1 is related to the partial pressure and concentration in Enclosure 2 via the interface sorption law:

\begin{equation}
C_1 = K P_2^n = K \left( C_2 RT \right)^n
\end{equation}

where $R$ is the ideal gas constant in J/mol/K, $T$ is the temperature in K, $K$ is the solubility, and $n$ is the exponent of the sorption law. For Henry’s law, $n=1$.

> ## Results

Reviewer: In this section, please add a note (see other cases with notes and warnings about typos) saying that we do not get the same kinetics as the results shown in TMAP7, which we attribute to a different diffusivity value.

> Two subcases are considered. In the first subcase, we assume that $K=1/RT$.

Reviewer: ```suggestion
Two subcases are considered. In the first subcase, we assume that $K=1/RT$ as is done in [!cite](ambrosek2008verification), which is expected to lead to $C_1 = C_2$ at equilibrium. In the second, $K=10/RT$, which is expected to lead to $C_1 = 10 C_2$. This second case is added to exercise TMAP8 in a case with a concentration jump.
```

> Consistent with the results from TMAP7, the pressure evolution in both enclosures is shown in [ver-1kb_comparison_time] as a function of time. Both pressure find equilibrium and become equal, which is consistent with $C_1 = K \cdot RT \cdot C_2^n$ for $K=1/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K \cdot R \cdot T=1$. As shown in [ver-1kb_mass_conservation], mass is conserved between the two enclosures over time.

Reviewer: ```suggestion
In the first subcase, consistent with the results from TMAP7, the pressure evolution in both enclosures is shown in [ver-1kb_comparison_time] as a function of time. Both pressures find equilibrium and become equal, which is consistent with $C_1 = K RT C_2^n$ for $K=1/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K R T=1$. As shown in [ver-1kb_mass_conservation], mass is conserved between the two enclosures over time, with a variation in mass of only xxx \%.
```

!media comparison_ver-1kb.py
       image_name=ver-1kb_comparison_time.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=ver-1kb_comparison_time
       caption=Equilibration of species pressures under Henry’s law for $K = 1/RT$.

!media comparison_ver-1kb.py
       image_name=ver-1kb_concentration_ratio.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=ver-1kb_concentration_ratio
       caption=Concentrations ratio between enclosures 1 and 2 at the interface for $K = 1/RT$. This verifies TMAP8's ability to apply the sorption law across the interface without a concentration jump.

!media comparison_ver-1kb.py
       image_name=ver-1kb_mass_conservation.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=ver-1kb_mass_conservation
       caption=Total mass conservation across both enclosures over time for $K = 1/RT$.

> In the second subcase, the sorption law with $K=10/RT$ does not lead to equal pressure in both enclosure. As illustrated in [ver-1kb_comparison_time_k10], the pressure jump maintains a ratio of $C_1/C_2 \approx 10$, which is consistent with the relationship $C_1 = K \cdot RT \cdot C_2^n$ for $K=10/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio_k10] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K \cdot R \cdot T=10$. Additionally, [ver-1kb_mass_conservation_k10] verifies that mass is conserved between the two enclosures over time.

Reviewer: ```suggestion
In the second subcase, the sorption law with $K=10/RT$ does not lead to equal pressure in both enclosure. As illustrated in [ver-1kb_comparison_time_k10], the pressure jump maintains a ratio of $C_1/C_2 \approx 10$, which is consistent with the relationship $C_1 = K RT C_2^n$ for $K=10/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio_k10] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K RT=10$. Additionally, [ver-1kb_mass_conservation_k10] verifies that mass is conserved between the two enclosures over time, with a variation in mass of only xxx \%..
```

# ver-1kb

# Henry’s Law Boundaries with No Volumetric Source

## General Case Description

Two enclosures are separated by a membrane that allows diffusion according to Henry’s law, with no volumetric source present. Enclosure 2 has twice the volume of Enclosure 1.

## Case Set Up

This verification problem is taken from [!cite](ambrosek2008verification).

> This setup describes a diffusion system in which tritium T$_2$ is modeled across a one-dimensional domain split into two enclosures. The total length of this domain is defined by the number of segments (20 segments) and the node size of $ 1.25\cdot 10^{-5}$ m, yielding a total system length of $ 2.5\cdot 10^{-4}$ m. The system operates at a constant temperature of $ 500$ Kelvin, with an ideal gas constant $ R=8.314$ J/(mol K). Initial tritium pressures are specified as $ 10^{5}$ Pa for Enclosure 1 and $ 10^{-10}$ Pa for Enclosure 2.

Reviewer: ```suggestion
This setup describes a diffusion system in which tritium T$_2$ is modeled across a one-dimensional domain split into two enclosures. The total length of this domain is defined by the number of segments (20 segments) and the node size of $1.25 \times 10^{-5}$ m, yielding a total system length of $2.5 \times 10^{-4}$ m. The system operates at a constant temperature of 500 Kelvin. Initial tritium pressures are specified as $10^{5}$ Pa for Enclosure 1 and $10^{-10}$ Pa for Enclosure 2.
```

The ideal gas constant is taken fromPhysicalConstant, so giving a different value here is inaccurate.

Reviewer: I think this line was causing the documentation to fail.

The diffusion process in each of the two enclosures can be described by

\begin{equation}
\frac{\partial C_1}{\partial t} = \nabla D \nabla C_1,
\end{equation}
and
\begin{equation}
\frac{\partial C_2}{\partial t} = \nabla D \nabla C_2,
\end{equation}

where $C_1$ and $C_2$ represent the concentration fields in enclosures 1 and 2 respectively, and $D$ denotes the diffusivity.

The concentration in Enclosure 1 is related to the partial pressure and concentration in Enclosure 2 via the interface sorption law:

\begin{equation}
C_1 = K P_2^n = K \left( C_2 RT \right)^n
\end{equation}

where $R$ is the ideal gas constant in J/mol/K, $T$ is the temperature in K, $K$ is the solubility, and $n$ is the exponent of the sorption law. For Henry’s law, $n=1$.

## Results

Two subcases are considered. In the first subcase, we assume that $K=1/RT$ as is done in [!cite](ambrosek2008verification), which is expected to lead to $C_1 = C_2$ at equilibrium. In the second, $K=10/RT$, which is expected to lead to $C_1 = 10 C_2$. This second case is added to exercise TMAP8 in a case with a concentration jump.

> In the first subcase, consistent with the results from TMAP7, the pressure evolution in both enclosures is shown in [ver-1kb_comparison_time] as a function of time. Both pressures find equilibrium and become equal, which is consistent with $C_1 = K RT C_2^n$ for $K=1/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K R T=1$. As shown in [ver-1kb_mass_conservation], mass is conserved between the two enclosures over time, with a variation in mass of only $3 \times 10^{-6}$ %.

Reviewer: ```suggestion
In the first subcase, consistent with the results from TMAP7, the pressure evolution in both enclosures is shown in [ver-1kb_comparison_time] as a function of time. Both pressures find equilibrium and become equal, which is consistent with $C_1 = K (RT C_2)^n$ for $K=1/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K R T=1$. As shown in [ver-1kb_mass_conservation], mass is conserved between the two enclosures over time, with a variation in mass of only $3 \times 10^{-6}$ %.
```

> In the first subcase, consistent with the results from TMAP7, the pressure evolution in both enclosures is shown in [ver-1kb_comparison_time] as a function of time. Both pressures find equilibrium and become equal, which is consistent with $C_1 = K RT C_2^n$ for $K=1/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K R T=1$. As shown in [ver-1kb_mass_conservation], mass is conserved between the two enclosures over time, with a variation in mass of only $3 \times 10^{-6}$ %.

Reviewer: How did you get the value of $3 \times 10^{-6}$ %? The percentage mass change (abs(max-min)/min*100) is close to 1%

!media comparison_ver-1kb.py
       image_name=ver-1kb_comparison_time.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=ver-1kb_comparison_time
       caption=Equilibration of species pressures under Henry’s law for $K = 1/RT$.

!media comparison_ver-1kb.py
       image_name=ver-1kb_concentration_ratio.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=ver-1kb_concentration_ratio
       caption=Concentrations ratio between enclosures 1 and 2 at the interface for $K = 1/RT$. This verifies TMAP8's ability to apply the sorption law across the interface without a concentration jump.

!media comparison_ver-1kb.py
       image_name=ver-1kb_mass_conservation.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=ver-1kb_mass_conservation
       caption=Total mass conservation across both enclosures over time for $K = 1/RT$.

> In the second subcase, the sorption law with $K=10/RT$ does not lead to equal pressure in both enclosure. As illustrated in [ver-1kb_comparison_time_k10], the pressure jump maintains a ratio of $C_1/C_2 \approx 10$, which is consistent with the relationship $C_1 = K RT C_2^n$ for $K=10/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio_k10] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K RT=10$. Additionally, [ver-1kb_mass_conservation_k10] verifies that mass is conserved between the two enclosures over time, with a variation in mass of only $10^{-6}$ %.

Reviewer: ```suggestion
In the second subcase, the sorption law with $K=10/RT$ does not lead to equal pressure in both enclosure. As illustrated in [ver-1kb_comparison_time_k10], the pressure jump maintains a ratio of $C_1/C_2 \approx 10$, which is consistent with the relationship $C_1 = K (RT C_2)^n$ for $K=10/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio_k10] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K RT=10$. Additionally, [ver-1kb_mass_conservation_k10] verifies that mass is conserved between the two enclosures over time, with a variation in mass of only $10^{-6}$ %.
```

> In the second subcase, the sorption law with $K=10/RT$ does not lead to equal pressure in both enclosure. As illustrated in [ver-1kb_comparison_time_k10], the pressure jump maintains a ratio of $C_1/C_2 \approx 10$, which is consistent with the relationship $C_1 = K RT C_2^n$ for $K=10/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio_k10] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K RT=10$. Additionally, [ver-1kb_mass_conservation_k10] verifies that mass is conserved between the two enclosures over time, with a variation in mass of only $10^{-6}$ %.

Reviewer: How did you get the percentage error again? Using approximate numbers for max and min, I get something closer to 0.25%.

## Results

Two subcases are considered. In the first subcase, it is assumed that the pressures in the two enclosures are in equilibrium, implying $K = 1/RT$.
Consistent with the results from TMAP7, the pressure evolution in both enclosures is shown in [ver-1kb_comparison_time] as a function of time, confirming the equilibrium between the pressures in enclosures 1 and 2. The linear regression in [ver-1kb_comparison_concentration] demonstrates that the concentration values at the interface comply with the sorption law, with a proportionality coefficient consistent with the solubility value $K = \approx 2.4 \times 10^{-4}$. As shown in [ver-1kb_mass_conservation], mass is conserved between the two enclosures over time.

!media comparison_ver-1kb.py
       image_name=ver-1kb_comparison_time.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=ver-1kb_comparison_time
       caption=Equilibration of species pressures under Henry’s law for $K=1/RT$.

!media comparison_ver-1kb.py
       image_name=ver-1kb_comparison_concentration.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=ver-1kb_comparison_concentration

>        caption=Concentration in enclosure 1 as a function of pressure in enclosure 2 at the interface for $K=1/RT$. Validation of the sorption law across the interface.

Reviewer: ```suggestion
       caption=Concentration in enclosure 1 as a function of pressure in enclosure 2 at the interface for $K=1/RT$. This verifies TMAP8's ability to apply the sorption law across the interface without a concentration jump.
```

Validation requires comparison against experimental data, verification is comparison against analytical solution. This here is a verification, not a validation.

In the second subcase, the sorption law with $K = 10/RT$ prevents the pressures in the two enclosures from reaching equilibrium. As illustrated in [ver-1kb_comparison_time_k10], the pressure jump maintains a ratio of $C_1/C_2 \approx 10$, which is consistent with the relationship $C_1 = K *RT * C_2$ for $K=10/RT$. The linear regression in [ver-1kb_comparison_concentration_k10] confirms that the concentration values at the interface adhere to the sorption law, with a proportionality coefficient aligned with the solubility value $K = \approx 2.4 \times 10^{-5}$. Additionally, [ver-1kb_mass_conservation_k10] verifies that mass is conserved between the two enclosures over time.

!media comparison_ver-1kb.py
       image_name=ver-1kb_comparison_time_k10.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=ver-1kb_comparison_time_k10
       caption=Pressures jump of species pressures under Henry’s law for $K=10/RT$.

!media comparison_ver-1kb.py
       image_name=ver-1kb_comparison_concentration_k10.png
       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
       id=ver-1kb_comparison_concentration_k10

>        caption=Concentration in enclosure 1 as a function of pressure in enclosure 2 at the interface for $K=10/RT$. Validation of the sorption law across the interface.

Reviewer: ```suggestion
       caption=Concentration in enclosure 1 as a function of pressure in enclosure 2 at the interface for $K=10/RT$. This verifies TMAP8's ability to apply the sorption law across the interface with a concentration jump..
```

This verification problem is taken from [!cite](ambrosek2008verification).

> This setup describes a diffusion system in which tritium T$_2$ is modeled across a one-dimensional domain split into two enclosures. The total length of this domain is defined by the number of segments (100 segments) and the node size of $1.25 \times 10^{-5}$ m, yielding a total system length of $2.5 \times 10^{-4}$ m. The system operates at a constant temperature of 500 Kelvin. Initial tritium pressures are specified as $10^{5}$ Pa for Enclosure 1 and $10^{-10}$ Pa for Enclosure 2.

Reviewer: You actually did not update the number of segments in the input file. 

Also, you should update the node size being documented here.

## Results

Two subcases are considered. In the first subcase, we assume that $K=1/RT$ as is done in [!cite](ambrosek2008verification), which is expected to lead to $C_1 = C_2$ at equilibrium. In the second, $K=10/RT$, which is expected to lead to $C_1 = 10 C_2$. This second case is added to exercise TMAP8 in a case with a concentration jump.

> In the first subcase, consistent with the results from TMAP7, the pressure evolution in both enclosures is shown in [ver-1kb_comparison_time] as a function of time. Both pressures find equilibrium and become equal, which is consistent with $C_1 = K RT C_2^n$ for $K=1/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K R T=1$. As shown in [ver-1kb_mass_conservation], mass is conserved between the two enclosures over time, with a variation in mass of only xxx \%.

Reviewer: Make sure to update `xxx` with the actual number.

## Results

Two subcases are considered. In the first subcase, we assume that $K=1/RT$ as is done in [!cite](ambrosek2008verification), which is expected to lead to $C_1 = C_2$ at equilibrium. In the second, $K=10/RT$, which is expected to lead to $C_1 = 10 C_2$. This second case is added to exercise TMAP8 in a case with a concentration jump.

> In the first subcase, consistent with the results from TMAP7, the pressure evolution in both enclosures is shown in [ver-1kb_comparison_time] as a function of time. Both pressures find equilibrium and become equal, which is consistent with $C_1 = K RT C_2^n$ for $K=1/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K R T=1$. As shown in [ver-1kb_mass_conservation], mass is conserved between the two enclosures over time, with a variation in mass of only $2 \cdot 10^{—6}$ \%.

Reviewer: ```suggestion
In the first subcase, consistent with the results from TMAP7, the pressure evolution in both enclosures is shown in [ver-1kb_comparison_time] as a function of time. Both pressures find equilibrium and become equal, which is consistent with $C_1 = K RT C_2^n$ for $K=1/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K R T=1$. As shown in [ver-1kb_mass_conservation], mass is conserved between the two enclosures over time, with a variation in mass of only $2 \times 10^{-6}$ %.
```

## Results

Two subcases are considered. In the first subcase, we assume that $K=1/RT$ as is done in [!cite](ambrosek2008verification), which is expected to lead to $C_1 = C_2$ at equilibrium. In the second, $K=10/RT$, which is expected to lead to $C_1 = 10 C_2$. This second case is added to exercise TMAP8 in a case with a concentration jump.

> In the first subcase, consistent with the results from TMAP7, the pressure evolution in both enclosures is shown in [ver-1kb_comparison_time] as a function of time. Both pressures find equilibrium and become equal, which is consistent with $C_1 = K (RT C_2)^n$ for $K=1/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K R T=1$. As shown in [ver-1kb_mass_conservation], mass is conserved between the two enclosures over time, with a variation in mass of only $1.0$ %.

Reviewer: The percentage value should be updated with the finer mesh.

Author: This is the percentage value obtained (1.0%) for `nb_segments_TMAP8=100` (to avoid the Exodus error)

       id=ver-1kb_mass_conservation
       caption=Total mass conservation across both enclosures over time for $K = 1/RT$.

> In the second subcase, the sorption law with $K=10/RT$ does not lead to equal pressure in both enclosure. As illustrated in [ver-1kb_comparison_time_k10], the pressure jump maintains a ratio of $C_1/C_2 \approx 10$, which is consistent with the relationship $C_1 = K RT C_2^n$ for $K=10/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio_k10] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K RT=10$. Additionally, [ver-1kb_mass_conservation_k10] verifies that mass is conserved between the two enclosures over time, with a variation in mass of only xxx \%..

Reviewer: Make sure to update `xxx` with the actual number.

       id=ver-1kb_mass_conservation
       caption=Total mass conservation across both enclosures over time for $K = 1/RT$.

> In the second subcase, the sorption law with $K=10/RT$ does not lead to equal pressure in both enclosure. As illustrated in [ver-1kb_comparison_time_k10], the pressure jump maintains a ratio of $C_1/C_2 \approx 10$, which is consistent with the relationship $C_1 = K RT C_2^n$ for $K=10/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio_k10] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K RT=10$. Additionally, [ver-1kb_mass_conservation_k10] verifies that mass is conserved between the two enclosures over time, with a variation in mass of only $8 \cdot 10^{—7}$ \%.

Reviewer: ```suggestion
In the second subcase, the sorption law with $K=10/RT$ does not lead to equal pressure in both enclosure. As illustrated in [ver-1kb_comparison_time_k10], the pressure jump maintains a ratio of $C_1/C_2 \approx 10$, which is consistent with the relationship $C_1 = K RT C_2^n$ for $K=10/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio_k10] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K RT=10$. Additionally, [ver-1kb_mass_conservation_k10] verifies that mass is conserved between the two enclosures over time, with a variation in mass of only $8 \times 10^{-7}$ %.
```

       id=ver-1kb_mass_conservation
       caption=Total mass conservation across both enclosures over time for $K = 1/RT$.

> In the second subcase, the sorption law with $K=10/RT$ does not lead to equal pressure in both enclosure. As illustrated in [ver-1kb_comparison_time_k10], the pressure jump maintains a ratio of $C_1/C_2 \approx 10$, which is consistent with the relationship $C_1 = K (RT C_2)^n$ for $K=10/RT$ and $n=1$. The concentration ratio between enclosures 1 and 2 in [ver-1kb_concentration_ratio_k10] shows that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K RT=10$. Additionally, [ver-1kb_mass_conservation_k10] verifies that mass is conserved between the two enclosures over time, with a variation in mass of only $0.25$ %.

Reviewer: The percentage value should be updated with the finer mesh.
